blockchain/source/contracts/ProfileRegistry.sol

```diff
@@ -133,4 +138,27 @@ contract ProfileRegistry {
             return IdentityLevel.ANONYMOUS;
         }
     }
+
+    function AddSonmRights(address _newSonm) onlyOwner public returns (bool) {
+        validators[_newSonm] = 0;
```

Comment:
Looks like the SONM validator has a `-1` value, but here you add _newSonm validator with anonymous level.